tsdb/head_test.go

```diff
@@ -1580,3 +1580,17 @@ func TestIsolationLowWatermarkMonotonous(t *testing.T) {
 	is.Close()
 	testutil.Equals(t, uint64(3), hb.iso.lowWatermark(), "After read has finished (iso state closed), low watermark should jump to three.")
 }
+
+func TestIsolationAppendIDZeroIsNoOp(t *testing.T) {
```

Comment:
```suggestion
func TestIsolationAppendIDZeroIsNoop(t *testing.T) {
```

Like in /storage/remote/read_test.go we usually use `Noop`.

Reply:
Done